MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/TitleView/ProgressBarView.swift

```diff
@@ -11,10 +11,26 @@ class ProgressBarView: UIView {
     var circularPath: UIBezierPath!
     var progressPath: UIBezierPath!
     
+    init() {
+        super.init(frame: .zero)
+        
+        endPoint = 0
+        createViews()
```

Comment:
I think you'd get the same result by calling `buildViews` here.
That way you only initialize constraints and colors once.